Cargo.toml

```diff
@@ -5,12 +5,12 @@ members = [
     "lightning-net-tokio",
 ]
 
-# Our tests do actual crypo and lots of work, the tradeoff for -O1 is well worth it
-[profile.test]
-opt-level = 1
-
+# Our tests do actual crypo and lots of work, the tradeoff for -O1 is well worth it.
+# Ideally we would only do this in profile.test, but profile.test seems to not be
+# used in testing, only profile.dev impacts the optimization level.
```

Comment:
Does the last paragraph in the following documentation describe the scenario?

https://doc.rust-lang.org/cargo/reference/profiles.html#profile-selection

Consider linking to it if so.

Reply:
Yes, it seems to. Sadly lto doesn't fix the issue either :(